dev-aws/kafka-shared-msk/cbc/cbc.tf

```diff
@@ -0,0 +1,16 @@
+resource "kafka_topic" "FraudEvents" {
+  name = "cbc.FraudEvents"
+
+  replication_factor = 3
+  partitions         = 15
+
+  config = {
+    "remote.storage.enable" = "true"
+    "retention.bytes"       = "-1"
+    "retention.ms"          = "-1"
+    "local.retention.ms"    = "3600000" # keep data locally for 1 hour
```

Comment:
I'd probably increase local retention to like a day to account for any consumer lag

Reply:
Although it's dev so it doesn't matter I guess, feel free to resolve

Reply:
fair point and maybe we should clarify with the help of the pubsub folks before doing this in prod. As you say, in dev it's probably pointless to do more than this!